config/ranks/corporate.txt

```diff
@@ -0,0 +1,50 @@
+Captain=Director
```

Comment:
I personally think Captain should be captain on both because that is how it is on all servers basically.

Reply:
Talk to @Vasily2013 about this one.

Reply:
Request Denied.

Further explanation: The director of the ship is the captain, captainship is a position, not a rank under the corporate rank structure. It's like saying that the "captain" of a passenger plane isn't actually a captain, he's a commercial pilot. The title is an honorific, not indicative of position other than job, which is still called in the codebase 'captain'.